### ISSUE TITLE
Harmless repeated nova warning for SR-IOV interfaces

### ISSUE THREAD

**Issue by lathiat** [canonical/hotsos#388]
The following warning is continuously logged by Nova for SR-IOV VMs everytime the periodic statistics taks runs once per minute

`2022-08-10 08:12:28.320 1728546 WARNING nova.pci.utils [req-880e81bf-1438-415a-b7c0-668169085494 - - - - -] No net device was found for VF 0000:31:11.7: nova.exception.PciDeviceNotFoundById: PCI device 0000:31:11.7 not found`

This looks fixed in Victoria and is harmless:
https://bugs.launchpad.net/nova/+bug/1808738

However I wondered about this message twice, and searching mattermost, at least 2-3 mentions of other people also wondering if it was normal. So I was thinking we could add a note about this 'harmless' message and we have also had other such harmless messages in the past and I wonder if this could be a kind of class of warning.. errors or warnings that are really harmless but maybe seem likely to be concerning.

**Comment by dosaboy** [comment]
agreed, i find it annoying when the agent-exceptions section is crammed with thousands of these and it make it more likely to miss other important exceptions when eyeballing the data. I'd vote for simply excluding the exception from the output.

**Comment by lathiat** [comment]
Problem is lots of people get confused and ask about it when later analysing manually. Search Mattermost for the string and you’ll see what I mean. 

I think some kind of harmless section would be helpful still.

**Comment by dosaboy** [comment]
Ah ok fair enough, i'll have a think about how to make that work.
